boards/posix/doc/bsim_boards_design.rst

```diff
@@ -242,13 +253,14 @@ which relies on the bs_trace API. Instead, for tracing the bs_trace API
 should be used directly.
 The same applies to other Zephyr APIs, including the entropy API, etc.
 
-posix_print backend
-===================
+posix_print and nsi_print backends
+==================================
 
 The bsim board provides a backend for the posix_print API which is expected by the posix ARCH
-and soc inf (POSIX) code.
-It simply routes the printk strings to the bs_trace bsim API.
-Any message printed to the posix_print API, which is also the default printk backend,
+and soc inf (POSIX) code, and for the nsi_print API expected by the native simulator.
```

Comment:
What is `nsi`? It's good practice to spell out abbreviations the first time you use them in documentation/descriptions

Reply:
I formatted the API names as code. In this case nsi_print_* are just the names of the functions in that API. This section is a minor note in the docs, which could be omitted all together. I do not think elaborating on why that API is named that way here would just derail things a bit too much.